Docker/startozwdaemon.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+shopt -s nocaseglob
+if [ ! -c "$USB_PATH" ]
+then
+	echo "USB Path \"$USB_PATH\" does not exist or is not a Character Device"
+	exit 128
+else
+	CMD="-s $USB_PATH"
+fi
+
+if [ ! -z "$MQTT_SERVER" ]
+then
+	CMD+=" --mqtt-server $MQTT_SERVER"
+	if [ ! -z "$MQTT_PORT" ]
+	then
+		CMD+=" --mqtt-port $MQTT_PORT"
+	else
+		CMD+=" --mqtt-port 1883"
+	fi
+fi
+	
+if [ -z "$STOP_ON_FAILURE" ]
+then
+	CMD+=" --stop-on-failure"
+else 
+	if [[ $STOP_ON_FAILURE != "false" ]]
+	then
+		CMD+=" --stop-on-failure"
+	fi
+fi
+
+if [ ! -z "$MQTT_TLS" ]
+then
+	if [[ $MQTT_TLS == "true" ]]
+	then
+		CMD+=" --mqtt-tls"
+	fi
+fi
+
+
+if [ -z "$OZW_INSTANCE" ] 
+then
+	CMD+=" --mqtt-instance 1"
+else 
+	CMD+=" --mqtt-instance $OZW_INSTANCE"
+fi
+
+if [ ! -z "$MQTT_USERNAME" ]
+then
+	CMD+=" --mqtt-username $MQTT_USERNAME"
+fi
+
+
+echo "Executing: /usr/local/bin/ozwdaemon $CMD"
+/usr/local/bin/ozwdaemon $CMD
```

Comment:
1. Consider using `exec /usr/local/bin/ozwdaemon $CMD` to drop the shell, together with `exec` form in the dockerfile, this would signals to be passed on to `ozwdaemon` instead of getting caught by the shell and not forwarded.
2. Pass on any argument passed to the script, `exec /usr/local/bin/ozwdaemon $CMD "$@"` as that would allow users add custom options to the `ozwdaemon` call. (e.g. `docker run openzwave/ozwdaemon:latest --help`) if exec form is used in the dockerfile.

Reply:
ok. point 2 is good as well.